Source/JavaScriptCore/b3/B3StackmapGenerationParams.cpp

```diff
@@ -47,14 +47,14 @@ RegisterSet StackmapGenerationParams::unavailableRegisters() const
     RegisterSet result = usedRegisters();
     
     RegisterSet unsavedCalleeSaves = RegisterSet::vmCalleeSaveRegisters();
-    unsavedCalleeSaves.exclude(m_context.code->calleeSaveRegisters());
+    unsavedCalleeSaves.exclude(m_context.code->calleeFullySavedRegisters());
 
     result.merge(unsavedCalleeSaves);
 
     for (GPRReg gpr : m_gpScratch)
-        result.clear(gpr);
+        result.ignore(gpr);
```

Comment:
really not sold at all on this API change. I think "clear" is a lot clearer when talking about a set than "ignore"